src/dataSources/booksDataSource.ts

```diff
@@ -0,0 +1,50 @@
+import { Book } from "../models/book";
+import * as fs from "fs";
```

Comment:
you don't need to import `* as fs` which just imports everything and names it `fs`. You can simply do that with
```suggestion
import fs from "fs";
```

Reply:
This applies for too all other places

Reply:
I did it, thanks for the advice!